cmd/ctr/commands/run/run.go

```diff
@@ -93,6 +93,10 @@ var ContainerFlags = []cli.Flag{
 		Name:  "pid-file",
 		Usage: "file path to write the task's pid",
 	},
+	cli.IntFlag{
+		Name:  "gpus",
+		Usage: "add gpus to the container",
```

Comment:
How about "add gpus to container, currently only nvidia gpus are supported" for clarity?

Reply:
No. Lets keep it generic so we don't have to change it back later